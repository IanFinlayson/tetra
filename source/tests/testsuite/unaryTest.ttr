def main():
   q = 100 ^ 50
   print("Just created q and q = ", q, "\n")
   w = 100 & 5
   print("Just created w and w = ", w, "\n")
   e = 222 | 222
   print("Just created e and e = ", e, "\n")
   x = not true
   print("Just created x and x = ", x, "\n")
   y = true
   print("Just created y and y = ", y, "\n")
   if not x:
      print("Not x.", "\n") 
      if not x and y:
         print("Not x and not y.", "\n") 
   print("End of main.", "\n")
