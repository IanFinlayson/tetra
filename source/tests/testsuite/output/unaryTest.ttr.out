Just created q and q = 86
Just created w and w = 4
Just created e and e = 222
Just created x and x = 0
Just created y and y = 1
Not x.
Not x and not y.
End of main.

